### DISCUSSION TITLE
Intel mac does not start

### DISCUSSION THREAD

**Discussion by quarties** [Try/OpenGothic#392] [Q&A] [answered]
Hey, I followed the latest install instruction, and when I run:
```
michalsypko@qrt-macbook opengothic % ./Gothic2Notr.sh -g /Users/michalsypko/gothic2
OpenGothic v1.0 dev
no *.ini file in path - using default settings
````

It just hangs. I see that process is working on G2 files (see screenshot), but I have no idea how to debug it further. Any help?
<img width="769" alt="image" src="https://user-images.githubusercontent.com/29309941/214640304-91c27aa9-35a5-47b4-b6d3-29ba2c8ba09f.png">

**Comment by Try** [comment]
Hi, @quarties and thanks for report!

Can you try to launch it withing debugger, so we can investigate callstack?
Command sequence should be like:
```
cd /path/to/Gothic2Notr
lldb ./Gothic2Notr
(lldb) process launch --environment DYLD_LIBRARY_PATH=.
..
# ctrl+c, once game stop responding
(lldb) bt all
```

**Comment by quarties** [comment, marked as answer]
Clean install from GOG made it work. Thanks for the help anyway!
